example/spec/helpers/workflowUtils.js

```diff
@@ -1,6 +1,23 @@
 'use strict';
 
 const get = require('lodash/get');
+const merge = require('lodash/merge');
```

Comment:
This probably warrants a changelog entry, particularly the common->spec/helpers move.